content/source/docs/cloud/getting-started/policies.html.md

```diff
@@ -1,19 +1,19 @@
 ---
 layout: "cloud"
-page_title: "Configuring Sentinel Policies - Getting Started - Terraform Cloud"
+page_title: "Enforcing Policies - Getting Started - Terraform Cloud"
 ---
 
-# Configuring Sentinel Policies
+# Enforcing Policies
```

Comment:
I found https://github.com/hashicorp/terraform-website/blob/master/content/source/docs/cloud/getting-started/runs.html.md#next-steps mentions `start applying policy checks`. Not sure if you wanted to enforce `enforce` there too.

Reply:
Fixed